app/src/main/res/layout/fragment_profile_details.xml

```diff
@@ -7,121 +7,166 @@
+                        android:layout_height="wrap_content"
+                        android:text="@string/name" />
 
-    <Button
-        android:id="@+id/fragment_profile_details_follow_button"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:text="@string/follow"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.0"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/fragment_profile_details_profile_image"
-        app:layout_constraintVertical_bias="0.97" />
+                    <RatingBar
+                        android:id="@+id/fragment_profile_details_profile_rating"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="13dp"
+                        app:layout_constraintEnd_toEndOf="parent" />
+                </LinearLayout>
+            </RelativeLayout>
 
-    <TextView
-        android:id="@+id/fragment_profile_details_profile_name"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="@string/name"
-        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_profile_rating"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintHorizontal_bias="0.412"
-        app:layout_constraintStart_toEndOf="@+id/fragment_profile_details_profile_image"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.607" />
+        </LinearLayout>
 
-    <RatingBar
-        android:id="@+id/fragment_profile_details_profile_rating"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="8dp"
-        android:layout_marginBottom="23dp"
-        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_profile_description"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toEndOf="@+id/fragment_profile_details_profile_image" />
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+                <Button
+                    android:id="@+id/fragment_profile_details_msg_button"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="6dp"
+                    android:layout_marginHorizontal="10dp"
+                    android:text="@string/send_message" />
```

Comment:
Here, in both button view, you could add the weight attribute, and set the `layout_width` to `0dp` so that the button share the equal amount of space in the `LinearLayout`

For both button you can add:
`android:layout_weight="0.5"`
`android:layout_width="0dp"`

Reply:
The asymmetry was intentional  =(